.bingo/README.md

```diff
@@ -10,4 +10,4 @@ This is directory which stores Go modules with pinned buildable package that is
 
 ## Requirements
 
-* Go 1.14+
+* Go 1.16+
```

Comment:
I don't think we need to change the min required version.

Reply:
thank you for your point out.